### ISSUE TITLE
Safari mute button has no focus visible styling and is out of sequence

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [microsoft/pxt-microbit#6207]
It's the tab stop before the stop button, but that's unexpected based on the layout and focus isn't visible so you feel lost tabbing through.

![Image](https://github.com/user-attachments/assets/a21be668-f216-485b-a539-ca300ce3479b)

I'm not sure if the placement when full screen is intended but it makes picking a sequence awkward as it can be before the board, partially over the board or to the right of the board depending on screen width:

![Image](https://github.com/user-attachments/assets/96157b7c-fbc6-4a45-91f4-dabeb6db3bfd)

**Comment by abchatra** [comment]
@microbit-matt-hillsdon I don't understand this issue. Can you elaborate? @riknoll do you have any idea?

**Comment by microbit-matt-hillsdon** [comment]
Clearer steps: 

1. Tab until the stop button in the sim is focussed
2. Shift-tab and notice there's no focus outline anywhere (you can reach this state tabbing forward too, just harder to pin down what's happened that way)
3. Press enter or query document.activeElement and realise it was the Safari mute button

The button should have a clear focus indicator (e.g. outline) and the tab position should be in a logical sequence with respect to other components on the board.

We sorted the tab order for other sim components in https://github.com/microsoft/pxt-microbit/pull/6113 but not this Safari specific one.
